src/main/java/net/minestom/server/item/ItemStack.java

```diff
@@ -159,6 +175,46 @@ public ItemStack read(@NotNull NetworkBuffer buffer) {
     @Contract(value = "_, -> new", pure = true)
     @NotNull ItemStack without(@NotNull DataComponent<?> component);
 
+    @Contract(value = "_, -> new", pure = true)
+    default @NotNull ItemStack withCustomName(@NotNull Component customName) {
+        return with(ItemComponent.CUSTOM_NAME, customName);
+    }
+
+    @Contract(value = "_, -> new", pure = true)
+    default @NotNull ItemStack withLore(@NotNull Component... lore) {
+        return with(ItemComponent.LORE, List.of(lore));
+    }
+
+    @Contract(value = "_, -> new", pure = true)
+    default @NotNull ItemStack withLore(@NotNull List<Component> lore) {
+        return with(ItemComponent.LORE, lore);
+    }
+
+    @Contract(value = "_ -> new", pure = true)
+    default @NotNull ItemStack withCustomModelData(int customModelData) {
+        return with(ItemComponent.CUSTOM_MODEL_DATA, customModelData);
+    }
+
+    @Contract(value = "_ -> new", pure = true)
+    default @NotNull ItemStack withGlowing(boolean glowing) {
+        return with(ItemComponent.ENCHANTMENT_GLINT_OVERRIDE, false);
```

Comment:
```suggestion
        return with(ItemComponent.ENCHANTMENT_GLINT_OVERRIDE, glowing);
```
Should this use the parameter value instead of `false`?
https://minecraft.wiki/w/Data_component_format#enchantment_glint_override

Reply:
Good catch, thanks